### DISCUSSION TITLE
2004/12/07/%E5%88%9A%E5%88%9A%E5%BC%80%E5%A7%8B%E5%BB%BA%EF%BC%8C%E6%B2%A1%E5%A4%9A%E5%B0%91%E6%97%B6%E9%97%B4%EF%BC%8C%E4%BB%A5%E5%90%8E%E5%86%85%E5%AE%B9%E4%BC%9A%E5%A4%9A%E7%9A%84%E3%80%82

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#2] [Announcements]
# 2004/12/07/%E5%88%9A%E5%88%9A%E5%BC%80%E5%A7%8B%E5%BB%BA%EF%BC%8C%E6%B2%A1%E5%A4%9A%E5%B0%91%E6%97%B6%E9%97%B4%EF%BC%8C%E4%BB%A5%E5%90%8E%E5%86%85%E5%AE%B9%E4%BC%9A%E5%A4%9A%E7%9A%84%E3%80%82

而且 msn space 还是 beta 版，好多功能都不好使。

https://blog.qizhen.xyz/2004/12/07/%E5%88%9A%E5%88%9A%E5%BC%80%E5%A7%8B%E5%BB%BA%EF%BC%8C%E6%B2%A1%E5%A4%9A%E5%B0%91%E6%97%B6%E9%97%B4%EF%BC%8C%E4%BB%A5%E5%90%8E%E5%86%85%E5%AE%B9%E4%BC%9A%E5%A4%9A%E7%9A%84%E3%80%82

**Comment by ruanqizhen** [comment]
之前博客下的留言：
```
枫
2009年05月18日 17:27
呵呵，眼睛真的不大呀
```
